stable/mariadb/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: mariadb
-version: 2.0.0
+version: 2.0.1
```

Comment:
Since this is feature addition it should likely be `2.1.0` to conform to [semver](http://semver.org).